contracts/libraries/OracleLibrary.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.5.0;
```

Comment:
my bad we should make this <0.8.0 as well because in 0.8.x math is safe by default